### DISCUSSION TITLE
When select double byte character in FEP(Japanese), title is completed too unexpectedly.

### DISCUSSION THREAD

**Discussion by ghost** [standardnotes/forum#2757] [General]
**Describe the bug**
When select double byte character in FEP(Japanese), title is completed too unexpectedly.
We use enter key to select double byte character in FEP and it seems affect Standard Note to finish editing the title.

**To Reproduce**
1. Go to 'title' editing
2. Starting entering double byte character using FEP(Japanese)
3. Select character in FEP to push enter key.
4. Character will be selected and title completed too unexpectedly.

**Expected behavior**
If using FEP and status is in selecting character, title must not be completed when return key is pushed.

**Screenshots**
<img width="675" alt="スクリーンショット 2020-06-10 11 36 55" src="https://user-images.githubusercontent.com/7712137/84220861-d898b600-ab0e-11ea-96fb-90af5bc3b842.png">
Go to editing title and entering double byte character by FEP(Front End word collecting Processor)

<img width="495" alt="スクリーンショット 2020-06-10 11 37 03" src="https://user-images.githubusercontent.com/7712137/84220946-02ea7380-ab0f-11ea-9ee4-0308f42fa0c3.png">
After select character in FEP and push enter key, then title was completed and cursor moved to note contents view.

**Desktop (please complete the following information):**
 - OS: Mac OS Catalina 10.15.5（19F101）
 - Standard Note version: 3.4.1

**Additional context**
I am using ATOK(the name of FEP).
I seems "Enter" key seems affect the phenomenon.

**Comment by JaspalSuri** [comment]
Hi, I'm sorry to hear that you're experiencing issues with Standard Notes when using the FEP. I'm afraid that since Standard Notes is a non-native app, there might not be much that the team can do to resolve the issue due to some technical limitations. 

Do you have a link to where I can download the software to it? Unfortunately I couldn't locate "ATOK FEP" on the web. 

Can you try to see if keyboard that is provided by the OS helps? Please see [here](https://github.com/standardnotes/forum/issues/926#issuecomment-610724756) for more information and please let me know if you have any questions!

**Comment by ghost** [comment]
Hi, Thanks for your reply.

I am using ATOK always but the issue can be reproduced when using the OS
default keyboard(FEP).

**Comment by ghost** [comment]
I am using ATOK always but the issue can be reproduced when using the OS default keyboard(FEP).

Keyboard settings are as below:

<img width="780" alt="スクリーンショット 2020-06-11 15 01 19" src="https://user-images.githubusercontent.com/7712137/84351732-e6296b00-abf6-11ea-9cd3-73fde81c276d.png">

Instead of using default keyboard, select sentence by entering return key caused completion of the title.

<img width="505" alt="スクリーンショット 2020-06-11 15 02 12" src="https://user-images.githubusercontent.com/7712137/84351766-f7727780-abf6-11ea-96ec-b8b1329f4e6e.png">

<img width="479" alt="スクリーンショット 2020-06-11 15 13 49" src="https://user-images.githubusercontent.com/7712137/84351786-fe00ef00-abf6-11ea-8502-acb658a8a70f.png">

App should keep editing title instead of completion.

**Comment by JaspalSuri** [comment]
Thank you so much for elaborating! I'll go ahead and forward this to our desktop and web app developer for review. :)

**Comment by ghost** [comment]
BTW, I tried same operation in Windows but the issue was not occurred so it seems the issue only for Mac platform.

**Comment by JaspalSuri** [comment]
Thank you for that piece of information! I will update my report with that!
